src/conn/conn_log.c

```diff
@@ -526,7 +526,7 @@ __log_prealloc_once(WT_SESSION_IMPL *session)
      */
     for (i = reccount; i < (u_int)conn->log_prealloc; i++) {
         WT_ERR(__wt_log_allocfile(session, ++log->prep_fileid, WT_LOG_PREPNAME));
-        WT_STAT_CONN_INCR(session, log_prealloc_files);
+        WT_STAT_CONN_INCRV(session, log_prealloc_files, (u_int)conn->log_prealloc - reccount);
```

Comment:
This is not correct. If the difference between these is that we'll allocate 10 log files, then this will increment the statistic by  100.

Reply:
And we don't want to pull this statistic out of the loop to set it at the end. If we allocate some and then get an error we want to update with the some we successfully allocated.